packages/devtools_app/lib/src/profiler/cpu_profile_service.dart

```diff
@@ -23,23 +22,16 @@ class CpuProfilerService {
     );
   }
 
-  /// Notifies that the vm profiler flag has changed.
-  ValueNotifier<Flag> get profilerFlagNotifier =>
-      serviceManager.vmFlagManager.flag(vm_flags.profiler);
-
-  ValueNotifier<Flag> get profileGranularityFlagNotifier =>
-      serviceManager.vmFlagManager.flag(vm_flags.profilePeriod);
-
-  Future clearCpuSamples() {
+  static Future clearCpuSamples() {
     return serviceManager.service
         .clearCpuSamples(serviceManager.isolateManager.selectedIsolate.id);
   }
 
-  Future<dynamic> setProfilePeriod(String value) {
+  static Future<dynamic> setProfilePeriod(String value) {
```

Comment:
nit: make these extension methods on serviceManager.service?
I don't think we have any services implemented as static methods so this is a little confusing and hides the fact that you are depending on the current serviceManager.service.